app/src/main/java/com/nervousfish/nervousfish/modules/cryptography/EncryptedSaver.java

```diff
@@ -0,0 +1,266 @@
+package com.nervousfish.nervousfish.modules.cryptography;
+
+
+import android.util.Base64;
+
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.UnsupportedEncodingException;
+import java.nio.charset.StandardCharsets;
+import java.security.InvalidAlgorithmParameterException;
+import java.security.InvalidKeyException;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.security.spec.InvalidKeySpecException;
+import java.util.Random;
+
+import javax.crypto.BadPaddingException;
+import javax.crypto.Cipher;
+import javax.crypto.IllegalBlockSizeException;
+import javax.crypto.NoSuchPaddingException;
+import javax.crypto.SecretKey;
+import javax.crypto.SecretKeyFactory;
+import javax.crypto.spec.PBEKeySpec;
+import javax.crypto.spec.PBEParameterSpec;
+
+/**
+ * Class used for encrypting strings.
+ * Commented out a bunch of methods in case of future needs.
+ */
+public final class EncryptedSaver {
+
+    private static final String PBE_WITH_MD5_AND_DES = "PBEWithMD5AndDES";
+    private static final Logger LOGGER = LoggerFactory.getLogger("Encrypted Saver");
+    private static final int SEED = 1234569;
+    private static final int IV_SPEC_SIZE = 8;
+    private static final String UTF_8 = "UTF-8";
+
+    /**
+     * Unused constructor for utility class.
+     */
+    private EncryptedSaver() {
+        // Prevent instantiations of QRGenerator
+
+    }
+
+    /**
+     * Generates a salt bytearray based on a seed and with a given length.
+     * @param seed  The seed to keep the salt consistent.
+     * @param length The length the salt should be.
+     * @return  The salt bytearray.
+     */
+    /*
```

Comment:
Yeah I mentioned this in the pr description, it's so that the working methods would still be there in case anyone needs them for later use